sunflower-land/sunflower-land
# [FIX] "Increase" seed hoarding limit

# Description

- fix issue where users can hit the hoarding limit for seeds while buying seeds with warehouse ready on island
- back end / contract changes might be needed
- I highly recommend core devs to fetch the hoarding limits from the contract and return it as an API for front end to consume, so the hoarding limits does not need to be hardcoded in 3 separate places.

## Type of change


# How Has This Been Tested?

- perform the following actions in order:
  - use up all seeds (any seed)
  - save data to blockchain
  - buy all seeds
  - if the issue is fixed, there should be no hoarding messages

# Checklist:
